server/public/model/property_value.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import "net/http"
+
+type PropertyValue struct {
+	ID         string `json:"id"`
+	TargetID   string `json:"target_id"`
+	TargetType string `json:"target_type"`
+	GroupID    string `json:"group_id"`
+	FieldID    string `json:"field_id"`
+	Value      string `json:"value"`
```

Comment:
Given that the `value` will be fairly free in the future (string, array, I also suggested a `number` type), would it be worth it to start right away with a `json.RawMessage` type for Value? Or are we gonna convert all values to "string" in the frontend and parse them based on the field type?

So instead of sending a `multiuser` as `[id1,id2,id3]`, we would stringify it as a CSV and do some parsing on the frontend?

Might be out of target for the POC, just thinking out loud.

Reply:
In other projects like boards we've used the string and parsing approach, but I like the idea of using generics for this. I'll explore it in a follow-up PR